### ISSUE TITLE
Specify shared extension properties

### ISSUE THREAD

**Issue by cholmes** [radiantearth/stac-spec#368]
There seems to be a growing desire to have properties that several domain-specific extensions used in a common area, potentially without prefixes. See #357 for some extensive discussion on this.

Ideally, I'd first like to see more domain specific extensions, with multiple implementations of each, before we start locking in what are the common ones. I'd like to try to ensure that are common definitions are truly common, and not tinged by a particular domain that makes them less usable by others. 

We are in decent shape, as we've got proposals in various states for:

* SAR #347 
* Labeled Training Data #362  
* Datacube #361 
* Pointcloud #157 

There's also open issues for:

* Drone data - #149 
* Full Motion Video - #156 

And of course we already have [EO](https://github.com/radiantearth/stac-spec/blob/master/extensions/eo/README.md). We also have the [time range extension](https://github.com/radiantearth/stac-spec/blob/master/extensions/datetime-range/README.md) which would be a very good candidate for a more general shared properties extension.

Putting this against 0.7.0, but that's hoping we get at least 5 of the content extensions in good shape so we are informed when we start extracting common properties.

**Comment by m-mohr** [comment]
Isn't this basically a bit broader issue as (or duplicate of?) #356?

**Comment by cholmes** [comment]
Ah, my bad, I missed that one. Will close this one, thanks!
